spec/factories.rb

```diff
@@ -470,6 +479,15 @@
     trait :universal_waiver do
       document_code { '999L' }
     end
+
+    trait :with_guidance do
+      guidace_cds { 'Guidance CDS' }
+      guidance_chief { 'Guidance CHIEF' }
+    end
+
+    initialize_with do
```

Comment:
So I _think_ you don't need this

Reply:
yep, we tried and we don't need it anymore.